### PR TITLE
feat(GitOpsCli): add var PREVIEW_ID_HASH_SHORT

### PR DESCRIPTION
**Pull request by christiansiegel**
Discussed in #163

### CODE REVIEW

**Review by joachimprinzbach — APPROVED** [review]

**Comment by joachimprinzbach** [comment]
Once this one is merged we can go for a new major release with the apiVersion 2?

File: gitopscli/gitops_config.py [hunk 1]
```diff
@@ -72,7 +73,8 @@ def __post_init__(self) -> None:
         assert isinstance(self.application_name, str), "application_name of wrong type!"
         assert isinstance(self.preview_host_template, str), "preview_host_template of wrong type!"
         self.__assert_variables(
-            self.preview_host_template, {"APPLICATION_NAME", "PREVIEW_ID_HASH", "PREVIEW_ID", "PREVIEW_NAMESPACE"}
+            self.preview_host_template,
+            {"APPLICATION_NAME", "PREVIEW_ID_HASH", "PREVIEW_ID_HASH_SHORT", "PREVIEW_ID", "PREVIEW_NAMESPACE"},
```
[the remark this answers is not in the document]
Comment (christiansiegel) [reply gitopscli/gitops_config.py:77]: yep, a bit of refactoring cannot hurt

**Comment by christiansiegel** [comment]
> Once this one is merged we can go for a new major release with the apiVersion 2?
 
Maybe you can test it first? Then we just have to change the `v2_beta` and update the docs.

**Pull request merged**